buku

```diff
@@ -395,6 +402,23 @@ class BukuCrypt:
             sys.exit(1)
 
 
+@total_ordering
+class SortKey:
+    def __init__(self, value, ascending=True):
+        self.value, self.ascending = value, bool(ascending)
+
+    def __eq__(self, other):
+        other = (other.value if isinstance(other, SortKey) else other)
+        return self.value == other
+
+    def __lt__(self, other):
+        other = (other.value if isinstance(other, SortKey) else other)
+        return self.value != other and ((self.value < other) == self.ascending)
+
+    def __repr__(self):
+        return ('+' if self.ascending else '-') + repr(self.value)
```

Comment:
This is used to apply a sort direction to each field individually (when sorting in Python)